src/query/service/src/auth.rs

```diff
@@ -27,13 +28,19 @@ use databend_common_users::JwtAuthenticator;
 use databend_common_users::UserApiProvider;
 use fastrace::func_name;
 
+use crate::servers::http::v1::TokenManager;
 use crate::sessions::Session;
 
 pub struct AuthMgr {
     jwt_auth: Option<JwtAuthenticator>,
 }
 
 pub enum Credential {
+    Databend {
```

Comment:
Databend -> SessionToken ?

Reply:
`Databend` auth/token refer a implementition/mechanism (bend-v1-xxx) (compare to JWT), whose **usage** can be a session token or refresh token for now.


or what about rename as DatabendSession， SessionRefeshToken and SessionAcessToken？

Reply:
the credential is represented by a Token here, so i think it's not a bad idea to name it with a "Token" suffix 👀

Reply:
in my opinion, we do not need putting the REFRESH TOKEN as a kind of credential. refresh token is only used in one renewing token api, it can be considered as a request parameter, instead of the identity credential (which is mostly the session token plays the role).

Reply:
> the credential is represented by a Token here, so i think it's not a bad idea to name it with a "Token" suffix 👀

done

Reply:
@flaneur2020 ci passed